packages/next/shared/lib/dynamic.tsx

```diff
@@ -131,7 +131,7 @@ export default function dynamic<P = {}>(
         )
       }
 
-      if (loadableOptions.loading != null) {
+      if (loadableOptions.loading != null && loadableOptions.loading !== undefined) {
```

Comment:
<img width="254" alt="image" src="https://user-images.githubusercontent.com/40715044/197381389-c7ac4d7e-dab9-4644-98e6-9ef4478d7f3a.png">

![image](https://user-images.githubusercontent.com/40715044/197381407-0a3d9c4e-bc75-4dfb-9ab7-019ddf154c9b.png)

Reply:
oh!

I thought that == will evaluate to true for all Falsy values...'' 

Live and learn.

So no PR at all, but then why it's happening.

Maybe it's fixed on canary.

Reply:
![imagen](https://user-images.githubusercontent.com/21086309/197461329-875839a0-2d78-4968-a906-4181d7365470.png)

We need it to support node v16

Reply:
Oh. I see what's going on.

We are setting a default `loadableOptions.loading` so when the user enables the `suspense` mode we need to unset the default.

It's not a critical bug, but developer experience matters, and people freak out with warnings with unknown origin. 

I have updated the PR.
I should have given it a more profound look, in the first place, instead of wasting your time.

Thanks for your revision.